### ISSUE TITLE
Paket restore large HTTP dependencies 

### ISSUE THREAD

**Issue by sergey-tihon** [fsprojects/Paket#2733]
### Description

I cannot restore large http dependencies (looks like 100s timeout) 
for https://github.com/sergey-tihon/Stanford.NLP.NET/issues/74

### Repro steps

run `mono .paket/paket.exe install`

```
http http://www.frijters.net/ikvmbin-8.1.5717.0.zip
http http://nlp.stanford.edu/software/stanford-corenlp-full-2017-06-09.zip
http https://nlp.stanford.edu/software/stanford-ner-2017-06-09.zip
http https://nlp.stanford.edu/software/stanford-parser-full-2017-06-09.zip
http https://nlp.stanford.edu/software/stanford-postagger-full-2017-06-09.zip
http https://nlp.stanford.edu/software/stanford-segmenter-2017-06-09.zip
```

### Expected behavior

zip archives downloaded and unzipped.

### Actual behavior

![2017-09-06_2220](https://user-images.githubusercontent.com/1197905/30130407-ab9342d8-9351-11e7-86e2-b8bf5ae97e5b.png)

### Known workarounds

No =( Maybe try to find some old version

**Comment by sergey-tihon** [comment]
This looks like default timeout of HttpClient https://msdn.microsoft.com/en-us/library/system.net.http.httpclient.timeout%28v=vs.110%29.aspx?f=255&MSPPError=-2147217396

**Comment by sergey-tihon** [comment]
fixed in 5.92.8
